src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -334,8 +338,9 @@ const useGroupInfo = () => {
   const handleMessageAnonymously = async () => {
     try {
       setIsLoadingInitChat(true);
-      await handleAnonymousMessage(selectedUser);
+      await handleAnonymousMessage(selectedUser, channelId, 'chat');
     } catch (e) {
+      console.log(e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the `handleMessageAnonymously` function could be improved. Currently, it just logs the error to the console. It would be better to provide some user feedback or handle the error in a way that doesn't potentially leave the user in an unknown state.

```diff
   const handleMessageAnonymously = async () => {
     try {
       setIsLoadingInitChat(true);
       await handleAnonymousMessage(selectedUser, channelId, 'chat');
     } catch (e) {
-      console.log(e);
+      // Show an error message to the user or handle the error appropriately.
     } finally {
       setIsLoadingInitChat(false);
       setOpenModal(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->